.pre-commit-config.yaml

```diff
@@ -77,4 +77,9 @@ repos:
   rev: '0.48'
   hooks:
   - id: check-manifest
-    stages: [manual]
+    args: [--no-build-isolation]
+
+ci:
+  autofix_prs: false
+  autoupdate_commit_msg: 'pre-commit autoupdate'
+  autoupdate_schedule: monthly
```

Comment:
How about quarterly? Sometimes some linter plugins can't keep up with the framework updates (I've seen this with flake8 + WPS) causing frequent pre-commit.ci update PRs to fail requiring partial reverts. This turned out annoying enough that I resorted to this solution in other projects.

Reply:
We can slow down further, if we want. I think monthly is fine for us (hence explicitly specifying it over the default of weekly).